2-mandatory/1-writers.js

```diff
@@ -70,8 +70,14 @@ Exercise 2:
   "Writer {firstName} {lastName} died at {age} years old."
 */
 
-function logDeadWritersInTheirForties() {
-  // write your code here
+function logDeadWritersInTheirForties( writers ) {
+  writers
+  .filter((writer) => writer.age <= 49 && !writer.alive)
```

Comment:
Good. This worsk with the set of writers in this exercise. But what would happen if one fo the wirters age was less than forty?